modules/ossm-release-notes-3-0-new-features.adoc

```diff
@@ -0,0 +1,114 @@
+
+For a list of supported component versions and support features, see "Service Mesh 3.0 feature support tables".
+
+For a complete list of changes between {SMProduct} 2 and {SMProduct} 3, see "Important information to know if you are migrating from OpenShift Service Mesh 2.6".
+
+[id="migration-guides_{context}"]
+== Migration guides
+
+This release adds a series of checklists and migration guides to help you migrate from {SMProduct} 2 to {SMProduct} 3.
+
+You must complete the checklists first. The checklists help you set up and configure {SMProduct} 2 and the `ServiceMeshControlPlane` resource to migrate to {SMProduct} 3 and the `Istio` control plane resource.
+
+Your migration depends on your deployment model:
+
+* Multitenant
+* Multitenant with cert-manager
+* Cluster-wide
+* Cluster-wide with cert-manager
+
+You can also migrate gateways.
+
+[id="new-istio-distribution-and-operator_{context}"]
+== New Istio distribution and operator
+
+{SMProduct} 3.0 is based on a Red{nbsp}Hat distribution of the link:https://istio.io/[Istio project] and is deployed with a new operator for Istio based on the Sail Operator project that is part of the istio-ecosystem organization. The Sail Operator includes a new set of custom resource definitions (CRDs) for managing Istio. For example, the `Istio` CRD replaces the `ServiceMeshControlPlane` CRD in previous releases of {SMProduct}.
+
+[id="support-for-istoctl-for-select-platforms-and-commands_{context}"]
+== Support for Istioctl for select platforms and commands
+
+This release adds support for select platforms and commands for Istioctl, the command line utility for the Istio project that includes many diagnostic and debugging utilities. For more information, see "Support for Istioctl".
+
+[IMPORTANT]
+====
+Installing Istio using the `istioctl` utility is not supported.
+====
+
+[id="support-for-multi-cluster_{context}"]
+== Support for multi-cluster
+
+This release introduces support for the Istio multi-cluster deployment models:
+
+* Multi-primary
+* Primary-remote
+* An external control plane
+
+The federation feature introduced in {SMProduct} 2.1 is not available in {SMPRoduct} 3.0.
+
+[id="multiple-control-planes-in-a-single-cluster_{context}"]
+== Multiple control planes in a single cluster
+
+This release adds support for the Istio feature of multiple control planes in a single cluster. This replaces the `MultiTenant` deployment model (mode) in {SMProduct} 2.
+
+[id="revision-based-upgrades_{context}"]
+== Revision based upgrades
+
+{SMProduct} 3 retains support for simple in-place style updates, and adds support for canary-style updates of the Istio control plane using Istio's revision feature.
+
+The `Istio` resource manages Istio revision labels using the `IstioRevision` resource. When the Istio resource's `updateStrategy` type is set to `RevisionBased`, it creates Istio revision labels using the Istio resource's name combined with the Istio version, for example `mymesh-v1-21-2`.
```

Comment:
`manages Istio revision labels using the` I don't think Istio is managing/creating revision labels. I'm not sure what was the idea here, probably to describe how the `Istiorevision` name is generated?